pkg/cmd/publish-provisional-artifacts/main.go

```diff
@@ -33,13 +33,15 @@ var provisionalReleasePrefixRE = regexp.MustCompile(`^provisional_[0-9]{12}_`)
 func main() {
 	var gcsBucket string
 	var outputDirectory string
+	var tagOverride string
 	var doProvisional bool
 	var isRelease bool
 	var doBless bool
 	flag.BoolVar(&isRelease, "release", false, "build in release mode instead of bleeding-edge mode")
 	flag.StringVar(&gcsBucket, "gcs-bucket", "", "GCS bucket")
 	flag.StringVar(&outputDirectory, "output-directory", "",
 		"Save local copies of uploaded release archives in this directory")
+	flag.StringVar(&tagOverride, "build-tag-override", "", "override the version from version.txt")
```

Comment:
Why drop the `build` in `tagOverride`? Asking because the word `tag` by itself is overloaded and explicitly saying `build tag` makes it clearer where the value will end up going.

Reply:
Fixed.